src/client/src/ui/spotTile/components/TileSwitch.tsx

```diff
@@ -22,7 +22,7 @@ const TileSwitch: React.SFC<Props> = ({
   currencyPair,
   spotTileData,
   executeTrade,
-  tornOff,
+  canPopout,
```

Comment:
I think `canPopout` is much better than `tornOff` — if you wrote RT fresh would you prefer `popout` over `tearOff`, `tearable` generally?